x-pack/plugins/infra/public/containers/ml/modules/metrics_hosts/module_descriptor.ts

```diff
@@ -48,24 +48,28 @@ const getJobIds = (spaceId: string, sourceId: string) =>
     {} as Record<MetricsHostsJobType, string>
   );
 
-const getJobSummary = async (spaceId: string, sourceId: string) => {
-  const response = await callJobsSummaryAPI(spaceId, sourceId, metricsHostsJobTypes);
+const getJobSummary = async (spaceId: string, sourceId: string, fetch: HttpHandler) => {
+  const response = await callJobsSummaryAPI(
+    { spaceId, sourceId, jobTypes: metricsHostsJobTypes },
+    fetch
+  );
   const jobIds = Object.values(getJobIds(spaceId, sourceId));
 
   return response.filter((jobSummary) => jobIds.includes(jobSummary.id));
 };
 
-const getModuleDefinition = async () => {
-  return await callGetMlModuleAPI(moduleId);
+const getModuleDefinition = async (fetch: HttpHandler) => {
+  return await callGetMlModuleAPI(moduleId, fetch);
 };
 
-const setUpModule = async (
-  start: number | undefined,
-  end: number | undefined,
-  datasetFilter: DatasetFilter,
-  { spaceId, sourceId, indices, timestampField }: ModuleSourceConfiguration,
-  partitionField?: string
-) => {
+const setUpModule = async (setUpModuleArgs: SetUpModuleArgs, fetch: HttpHandler) => {
+  const {
+    start,
+    end,
+    moduleSourceConfiguration: { spaceId, sourceId, indices, timestampField },
```

Comment:
missing param here: `datasetFilter: DatasetFilter,`. It's unused, but unfortunately, it's apart of the signature.